library/src/noop/java/com/telefonica/androidlogger/domain/AppLogger.kt

```diff
@@ -4,9 +4,11 @@ import android.content.Context
 
 @Suppress("UNUSED_PARAMETER")
 @JvmOverloads
-fun initAppLogger(context: Context, logCategories: List<LogCategory> = emptyList()) {
-
-}
+fun initAppLogger(
+    context: Context,
+    logCategories: List<LogCategory> = emptyList(),
+    logToDisk: Boolean = true
+) {}
```

Comment:
That will leave a warning in sonarQube unless the body has something like a comment:
```
{
    /* Do nothing */
}
```